templates/_layout.html

```diff
@@ -46,6 +46,14 @@
         'title': 'Emergency Alerts',
         'columns': 2,
         'items': [
+          {
+            'text': 'Past alerts',
+            'href': '/alerts/past-alerts'
+          },
+          {
+            'text': 'Current alerts',
+            'href': '/alerts/current-alerts'
+          },
```

Comment:
Current should come before past, to match the prototype (and because stuff on the web is typically ordered reverse chronologically).

Other than this I think the PR is good to go. I’ve spun it up locally and had a good click around.

```suggestion
          {
            'text': 'Current alerts',
            'href': '/alerts/current-alerts'
          },
          {
            'text': 'Past alerts',
            'href': '/alerts/past-alerts'
          },
```

Reply:
OK. I've amended the commit to reverse the ordering.